services/federator/test/unit/Test/Federator/Client.hs

```diff
@@ -14,6 +14,7 @@
 --
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
+{-# LANGUAGE LambdaCase #-}
```

Comment:
this is in `/package-defaults.yaml` and should need to be added in the module.  i don't mind the noise (in this case), just saying.

Reply:
Good spot :+1: 

This is now fixed.